cmd/utils/flags.go

```diff
@@ -840,20 +825,19 @@ func MustMakeChainConfigFromDb(ctx *cli.Context, db ethdb.Database) *core.ChainC
 	glog.V(logger.Warn).Info(fmt.Sprintf("%v blockchain hard-forks associated with this genesis block:", len(c.Forks)))
 
 	netsplitChoice := ""
-	testnet := []string{"Homestead", "GasReprice"} //TODO create separate chain config for testnet, then remove that
-	sort.Strings(testnet)
-
 	for i := range c.Forks {
 		if c.Forks[i].NetworkSplit {
 			netsplitChoice = fmt.Sprintf("resulted in a network split (support: %t)", c.Forks[i].Support)
 		} else {
 			netsplitChoice = ""
 		}
-		testnetEnabled := sort.SearchStrings(testnet, c.Forks[i].Name) < len(testnet) && testnet[sort.SearchStrings(testnet, c.Forks[i].Name)] == c.Forks[i].Name
-		if !ctx.GlobalBool(TestNetFlag.Name) || testnetEnabled {
-			glog.V(logger.Warn).Info(fmt.Sprintf(" %v hard-fork at block %v %v", c.Forks[i].Name, c.Forks[i].Block, netsplitChoice))
+		pad := ""
+		if big.NewInt(1000000).Cmp(c.Forks[i].Block) > 0 {
+			pad = " "
```

Comment:
This is a little excessive, maybe cleaner to just have the logger leave an extra space